README.md

```diff
@@ -77,17 +77,22 @@ The URL for the Schaufensterkonditionen is:
 
 ### Query Top-Schaufensterkondition
 
+#### Hints
+
+* The calculation of Top-Schaufensterkondition requires a valid `auszahlungsbetrag` and `laufzeitInMonaten`.
+  If the provided values are not valid, an [GraphQL-Error](#graphql-errors) with the status code `400` is returned.
+
 #### Request
 
 The GraphQL-Query is called `topSchaufensterkondition` and contains the following parameter:
 
-| Parameter Name     | Type                                      | Default Value                                     |
-|--------------------|-------------------------------------------|---------------------------------------------------|
-| partnerId          | [Partner-ID](#partner-id)                 | The Partner-ID of the API Client                  |
-| auszahlungsbetrag  | Euro!                                     | - (mandatory field)                               | 
-| laufzeitInMonaten  | Int                                       | -                                                 | 
-| finanzierungszweck | [Finanzierungszweck](#finanzierungszweck) | Calculation over all values of Finanzierungszweck |
-| datenkontext       | [Datenkontext](#datenkontext)             | `TESTUMGEBUNG`                                    |
+| Parameter Name     | Type                                       | Default Value                                     | Condition      |
+|--------------------|--------------------------------------------|---------------------------------------------------|----------------|
+| partnerId          | [Partner-ID](#partner-id)                  | The Partner-ID of the API Client                  |                |
+| auszahlungsbetrag  | Euro!                                      | - (mandatory field)                               | greater than 0 | 
+| laufzeitInMonaten  | Int                                        |                                                   | greater than 0 |
```

Comment:
Even if the datatype is doesn't make it mandatory, we should mark it as mandatory. And I would make the additional text more verbose, just like the rest.
```suggestion
| laufzeitInMonaten  | Int                                        | - (mandatory field)                               | Value must be greater than 0 |
```